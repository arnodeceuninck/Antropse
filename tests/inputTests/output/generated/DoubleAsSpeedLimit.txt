De snelheid is geen integer
