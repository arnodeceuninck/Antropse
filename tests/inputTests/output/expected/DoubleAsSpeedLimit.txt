Partial Import: Ongeldige informatie bij de snelheidslimiet van de weg
